controllers/comments.controllers.js

```diff
@@ -0,0 +1,13 @@
+const { selectCommentsForArticle } = require("../models/comments.models")
+const { checkArticleExists } = require("../models/articles.models")
+
+exports.getCommentsForArticle = (req, res, next) => {
+    const { article_id } = req.params
+    Promise.all([selectCommentsForArticle(article_id), checkArticleExists(article_id)])
```

Comment:
if the article doesn't exist, wouldn't it throw a psql error in the selectCommentsforArticle?